NDB.Covid19/NDB.Covid19.iOS/Views/DailyNumbers/DailyNumbersViewController.cs

```diff
@@ -145,5 +151,26 @@ private void SetStyling()
 			NavigationController.PopViewController(true);
 			NavigationHelper.GoToResultPage(this, false);
 		}
+
+		private void SetupSubTextWithLink(string text, UITextView textView)
+		{
+			// Necessary to unify horizontal alignment with the rest of the text on the page
+			textView.TextContainerInset = UIEdgeInsets.Zero;
+			textView.TextContainer.LineFragmentPadding = 0;
+
+			//Defining attibutes inorder to format the embedded link
+			NSAttributedStringDocumentAttributes documentAttributes = new NSAttributedStringDocumentAttributes { DocumentType = NSDocumentType.HTML };
+			documentAttributes.StringEncoding = NSStringEncoding.UTF8;
+			NSError error = null;
+			NSAttributedString attributedString = new NSAttributedString(NSData.FromString(text, NSStringEncoding.UTF8), documentAttributes, ref error);
+
+			//Ensuring text is resiezed correctly when font size is increased
+			StyleUtil.InitTextViewWithSpacingAndUrl(textView, StyleUtil.FontType.FontRegular, attributedString, 1.28, 16, 22);
+
+			textView.TextColor = ColorHelper.TEXT_COLOR_ON_BACKGROUND;
+
+			//ForegroundColor sets the color of the links. UnderlineStyle determins if the link is underlined, 0 without underline 1 with underline.
+			textView.WeakLinkTextAttributes = new NSDictionary(UIStringAttributeKey.ForegroundColor, ColorHelper.TEXT_COLOR_ON_BACKGROUND, UIStringAttributeKey.UnderlineStyle, new NSNumber(1));
+		}
```

Comment:
Isn't this already part of some Utils class?

Reply:
was inspired by what was done on settings page 4, which uses a method placed in SettingsContextTextView. Otherwise I can see that we have a method called InitUITextViewWithSpacingAndUrl in styleutil, that is never used though, could perhaps adjust this to be used instead?